.github/workflows/git-secrets.yaml

```diff
@@ -4,7 +4,7 @@ jobs:
   gitleaks-scan:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
Gitleaks had a warning for using v2 being old so I just changed it to v3.  Documentation for gitleaks github action also has it using v3 currently.